### DISCUSSION TITLE
Lots of small files causing slow scans?

### DISCUSSION THREAD

**Discussion by liujfeng** [minio/minio#15490] [Q&A] [answered]
I use minio to store 100 million data, the size is between 20k ~ 512k. The statistics obtained with `mc admin info` are much smaller than the actual capacity. Also setting `ilm` doesn't work either. Is this due to the slow scan of a large number of small file data? Have a good suggestion?

**Comment by harshavardhana** [comment, marked as answer]
@liujfeng this simply looks like a deployment issue to me - https://min.io/pricing - please subscribe for us to debug your architecture issues.

**Comment by klauspost** [comment]
`mc admin scanner info ALIAS` provides scanner statistics. If you need more help, follow the link above.
